src/api/load.js

```diff
@@ -33,6 +34,26 @@ export type LoadOptions = {|
   +githubToken: ?GithubToken,
 |};
 
+export async function loadContext(
+  options: LoadOptions,
+  reporter: TaskReporter
+): Promise<void> {
+  const {
+    sourcecredDirectory,
+    githubToken,
+    project,
+    params,
+    weightsOverrides,
+  } = options;
+  const data = new DataDirectory(sourcecredDirectory);
+  const context = new LoadContext({cache: data, githubToken, reporter});
+  const result = await context.load(project, {
+    params: params || {},
```

Comment:
(nit): shouldn't params be pass thru?

Reply:
This time we have, the problem:
https://github.com/sourcecred/sourcecred/blob/c61d356dc2bed3f44e2d006349992e05b0ab8bc5/src/api/load.js#L28-L30
And the workaround in the same file :sweat_smile:
A direct pass through would cause a Flow error.

Currently I'm using this `loadContext` function as a drop-in replacement of the legacy `load`. After all legacy stuff is removed we could update the API to be more natural here.